system/View/Traits/DecoratesViews.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace CodeIgniter\View\Traits;
+
+use CodeIgniter\View\Exceptions\ViewException;
+use CodeIgniter\View\Interfaces\ViewDecorator;
+
+use function config;
+
+trait DecoratesViews
+{
+    /**
+     * Runs the generated output through and declared
+     * view decorators.
+     *
+     * @param string $html
+     *
+     * @return string
+     */
+    private function decorateOutput(string $html): string
```

Comment:
Still asking questions... If anyone has their own `RendererInterface` service wouldn't it likely be an extension of `View`? If so it might make sense to have this method accessible. We could accomplish the same protection with `final protected function decorateOutput()`.

Reply:
It doesn't like that when it's in a trait, during testing at least. Complained about not being able to override the function. Looks like either private or protected. MOving to protected for now.